### PR TITLE
qemu version change

### PR DESCRIPTION
**Pull request by md-shafiul-alam**
# Description
version 9.0.1 was removed, using a slightly older version to avoid frequent update.

Changes proposed in this pull request:
-
-
-

### CODE REVIEW

**Comment by napetrov** [comment]
@keeranroth  @rakshithgb-fujitsu  - please review/comment

**Comment by rakshithgb-fujitsu** [comment]
Looks fine to me. Some CI cases are failing though.

File: .ci/env/apt.sh [hunk 1]
```diff
@@ -63,7 +63,7 @@ function install_qemu_emulation_apt {
 }
 
 function install_qemu_emulation_deb {
-    qemu_deb=qemu-user-static_9.0.1+ds-1~bpo12+1_amd64.deb
+    qemu_deb=qemu-user-static_8.2.4+ds-1_amd64.deb
```
Comment (keeranroth) [inline .ci/env/apt.sh:66]: This has been flaky in the past as well. Let's pass a few version numbers through, and get the first one that exists. Something like the following seems to work (feel free to tidy up / change as necessary

```
set +e

versions=(9.0.2 9.0.1 8.2.4)
found_version=""
for version in ${versions[@]}; do
    qemu_deb="qemu-user-static_${version}+ds-1_amd64.deb"
    echo "Checking for http://ftp.debian.org/debian/pool/main/q/qemu/${qemu_deb}"
    if wget -q --method=HEAD http://ftp.debian.org/debian/pool/main/q/qemu/${qemu_deb} &> /dev/null;
    then
        echo "Found qemu version ${version}"
        found_version=${qemu_deb}
        break
    fi
done

set -eo pipefail
if [[ -z "${found_version}" ]] ; then
    # If nothing is found, error out and fail
    echo "No qemu found with version numbers ${versions[@]}"
    false
fi

wget http://ftp.debian.org/debian/pool/main/q/qemu/${found_version}
sudo dpkg -i ${found_version}
```

**Review by keeranroth — CHANGES_REQUESTED** [review]
Maybe try adding a list of packages, so that if one goes away for a few days, we have a backup

**Review by keeranroth — APPROVED** [review]
Looks good to me. I don't think that the failures in the other pipelines are related, from a quick look

**Comment by md-shafiul-alam** [comment]
> Looks good to me. I don't think that the failures in the other pipelines are related, from a quick look

agree, thanks for the great suggestion

**Review by ethanglaser — APPROVED** [review]

**Comment by ethanglaser** [comment]
LinuxSklearnex fail will be addressed by https://github.com/intel/scikit-learn-intelex/pull/1982

**Pull request merged**